bahatii/TeamWork
# #168828491 built out necessary frameworks and dependancies to set up the…

#### What does this PR do?
adds necessary framework and dependecies to the directory to build th APIs
#### Description of Task to be completed?
was the following goals achieved:
was express installed propery
was packege.json well installed
was babel and nodemon installed
was moment and UUID  dependacy installed
was Eslint- Airbnb-style installed properly
was 

#### How should this be manually tested?
open the project in vs code then run NPM start, then go to the postman and send a get request with localhost:/3000 then you should see the welcome message
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#168828491
#### Questions:
